File: 程式語言 V2答案檔/Go/701.go
package main
import (
	"fmt"
	"math"
)

func main(){
	var a, b, c float64
	fmt.Scan(&a)
	fmt.Scan(&b)
	fmt.Scan(&c)
	s := float64(a+b+c)/2.0
	result := math.Sqrt(s * (s-a) * (s-b) * (s-c))
	fmt.Printf("%.2f\n", result)
}

File: 程式語言 V2答案檔/PHP/701.php
<?php
    $a = intval(fgets(STDIN));
    $b = intval(fgets(STDIN));
    $c = intval(fgets(STDIN));
    $s = ($a+$b+$c)/2.0;
    $result = sqrt($s*($s-$a)*($s-$b)*($s-$c));
    printf("%.2f\n", $result);
?>